docs/src/main/paradox/openshift.md

```diff
@@ -51,24 +59,17 @@ Container Registry in Lightbend's OpenShift cluster, CentralPark. You will have
 ```bash
 sbt docker:publishLocal
 
-export DOCKER_REGISTRY_SERVER=docker-registry-default.centralpark.lightbend.com
-export DOCKER_REGISTRY=$DOCKER_REGISTRY_SERVER/$OPENSHIFT_PROJECT
-
 docker login -p $TOKEN -u unused $DOCKER_REGISTRY_SERVER
 docker tag $IMAGE:$TAG $DOCKER_REGISTRY/$IMAGE:$TAG
 docker push $DOCKER_REGISTRY/$IMAGE:$TAG
-```
 
-Apply your deployment : 
-
-```bash
-$ oc apply -f deployment/openshift-play-scala-grpc-example.yml
-deployment.apps "play-scala-grpc-example-v1-0-snapshot" configured
-service "play-scala-grpc-example" unchanged
-route.route.openshift.io "play-scala-grpc-route" created
+## The `kustomize` step uses a `kustomization.yml` prepared for $DOCKER_REGISTRY/$IMAGE:$TAG. You will have to
+## create your own `deployment/overlays` folder (make a copy of `deployment/overlays/centralpark` and edit
```

Comment:
Can we call this place `deployment/overlays/openshift` instead?

Reply:
I wanted to, but the `kustomization.yml` is specific to `centralpark` since it actually sets the registry for central park. The whole point of this overlay is to be centralpark specific.

I had removed all references to centralpark and when I got here I realised that using the right balance between `openshift` and `centralpark` references in the docs was actually exemplifying quite well what to do and how to do it. What I mean is that it was easy for me to talk about `openshift` when it was not specific to a cluster and then shift to `centralpark` when it was something meant for the user to switch. I think keeping `centralpark` will make it easy for a user to locate and edit when they want to test this on their cluster.